lightly/api/api_workflow_datasets.py

```diff
@@ -7,6 +7,23 @@
 
 class _DatasetsMixin:
 
+    @property
+    def dataset_type(self) -> str:
+        """Returns the dataset type of the current dataset.
+        
+        """
+        dataset = self._get_current_dataset()
+        return dataset.type # type: ignore
+
+    def _get_current_dataset(self):
+        """Returns the dataset with id == self.dataset_id.
+        
+        """
+        dataset_id = self.dataset_id
+        datasets: List[DatasetData] = self._datasets_api.get_datasets()
+        datasets = [dataset for dataset in datasets if dataset.id == dataset_id]
+        return datasets[0]
```

Comment:
Maybe use next instead of the first item from the list comprehension? 
see https://stackoverflow.com/questions/9868653/find-first-sequence-item-that-matches-a-criterion

Reply:
Ah good catch, thanks! Changed it :)